lib/routes/install.js

```diff
@@ -41,13 +41,13 @@ module.exports = app => {
 };
 
 const generatorMatchTerm = (generator, term) => `${generator.name} ${generator.description}`.indexOf(term) !== -1;
-const getAllGenerators = _.memoize(() => npmKeyword('yeoman-generator'));
+const getGenerators = _.memoize(term => npmKeyword(['yeoman-generator', term]));
```

Comment:
Should we still re-filter by ourselves later on then?

`generatorMatchTerm(generator, term);`

Reply:
Uhm. No. Guess it was a little late last night 🙈 
Going to remove that.